vendor/travis-core/lib/travis/model/user.rb

```diff
@@ -17,7 +17,7 @@ class User < Travis::Model
 
   before_create :set_as_recent
   after_create :create_a_token
-  after_commit :sync, on: :create
+  # after_commit :sync, on: :create
```

Comment:
Could we delete instead of comment? I'm generally in favour of using git to recall old code, where necessary. Maybe you're unsure if these things can be permanently removed?